crowndata-evaluation/crowndata_evaluation/services/action_consistency/clustering.py

```diff
@@ -1,6 +1,7 @@
 import numpy as np
 from typing import Dict
 import sklearn.cluster
+from scipy.spatial import cKDTree  # or sklearn.neighbors import KDTree
```

Comment:
remove unnecessary comments?

Reply:
- [ ] compare Scipy.spatial and sklearn.neighbors KDTree